### ISSUE TITLE
When will the training scripts and related code of ViCLIP be released?

### ISSUE THREAD

**Issue by XuecWu** [OpenGVLab/InternVideo#85]
As described above.
Looking forward to your reply.

**Comment by Arseny5** [comment]
> As described above. Looking forward to your reply.
Same question) Dear team InternVid, please can you share training scripts? And one more question, what loss do you use for train ViClip? Contrastive (like vanilla CLIP) or BCE?

**Comment by Arseny5** [comment]
> As described above. Looking forward to your reply.

Same question) Dear team InternVid, please can you share training scripts? And one more question, what loss do you use for train ViClip? Contrastive (like vanilla CLIP) or BCE?

**Comment by shepnerd** [comment]
FYI. https://github.com/OpenGVLab/InternVideo/tree/main/Pretrain/ViCLIP
